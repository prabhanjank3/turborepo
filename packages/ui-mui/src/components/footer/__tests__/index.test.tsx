import * as React from 'react';
import { render } from '@testing-library/react';

import Footer from '..';

describe('<Footer  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Footer />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
