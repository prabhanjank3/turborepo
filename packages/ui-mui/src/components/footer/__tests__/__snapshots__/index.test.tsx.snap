// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer  /> should match snapshot 1`] = `
<div
  class="MuiBox-root css-12eyfu8"
>
  <p
    class="MuiTypography-root MuiTypography-body1 css-plxji9-MuiTypography-root"
  >
     
    Â© 2024 Private Journal - Developed by Prabhanjan Kulkarni
  </p>
</div>
`;
